eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/jdbc/statements/TimestampWriter.java

```diff
@@ -0,0 +1,10 @@
+package org.axonframework.eventsourcing.eventstore.jdbc.statements;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.time.Instant;
+
```

Comment:
I am missing class level and method javadoc here.
Don't forget to add a since and author tag to the class level javadoc too.

Reply:
Added javadoc.